### PR TITLE
sdk,cli: expose contracts info in SDK and provide /contracts endpoint in CLI

### PR DESCRIPTION
**Pull request by andrevmatos**
Part of #1692 

**Short description**
Implements `/contracts` endpoint. Still undocumented on spec, but offered by python API.

**Definition of Done**

- [x] Steps to manually test the change have been documented
- [x] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1.
2.

### CODE REVIEW

File: raiden-cli/src/routes/api.v1.ts [hunk 1]
```diff
@@ -19,15 +19,24 @@ export function makeApiV1Router(this: Cli): Router {
...
 
   router.get('/version', (_request: Request, response: Response) => {
+    response.json({ version: Raiden.version });
+  });
+
+  router.get('/contracts', (_request: Request, response: Response) => {
+    const contracts = this.raiden.contractsInfo;
     response.json({
-      version: Raiden.version,
+      contracts_version: Raiden.contractVersion,
```
Comment (weilbith) [inline raiden-cli/src/routes/api.v1.ts:28]: Similar to an earlier issue. We should maybe consider to remove old interface definitions for API objects. Or if you still see an advantage of them add one for this route endpoint as well.

File: raiden-cli/src/routes/api.v1.ts [hunk 2]
```diff
@@ -19,15 +19,24 @@ export function makeApiV1Router(this: Cli): Router {
...
+      monitoring_service_address: contracts.MonitoringService.address,
+      one_to_n_address: '',
     });
   });
 
   router.get('/address', (_request: Request, response: Response) => {
-    response.json({
-      our_address: this.raiden.address,
-    });
+    response.json({ our_address: this.raiden.address });
```
Comment (weilbith) [inline raiden-cli/src/routes/api.v1.ts:39]: Has formatting changed? I hate it when formatting is not deterministic. :expressionless:

Comment (andrevmatos) [reply raiden-cli/src/routes/api.v1.ts:39]: It didn't change, but prettier's recommended rules allow small objects to be either declared in one line or split lines. I just wanted to reduce the line counter for something so simple. If it becomes bigger, with more properties making it cross 99char line limit, then it must be broken.

File: raiden-cli/src/routes/api.v1.ts (hunk 1 above)
Comment (andrevmatos) [reply raiden-cli/src/routes/api.v1.ts:28]: As I replied there, let's decide on this and do it on a separate PR.

**Pull request merged**